.github/workflows/release.yml

```
@@ -4,10 +4,10 @@ on:
   schedule:
     - cron: '19 3 * * *'
   push:
-    branches: [ main, 15-3.3, 17-3.5 ]
+    branches: [ main, 15-3.3, 18-3.6 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)